src/core.js

```diff
@@ -27,7 +27,7 @@ define( [
 "use strict";
 
 var
-	version = "@VERSION",
+	version = '@VERSION',
```

Comment:
format: style mismatch:
`fe1ba627` Replace ⏎⇥⁺ with ␣ in the beginning of the line